sdks/python/apache_beam/examples/cookbook/ordered_window_elements/streaming.py

```diff
@@ -89,7 +90,7 @@ def __init__(
       self,
       duration: DurationTypes,
       slide_interval: DurationTypes,
-      offset: TimestampTypes,
+      offset: DurationTypes,
```

Comment:
looks like a break change to me. Could we keep both like `offset: Union[TimestampTypes, DurationTypes]`?